docs/program/_index.md

```diff
@@ -293,6 +293,10 @@ This connection must be established for your program to be executed properly on
 
 {{< readfile "/static/include/snippet/secret-share.md" >}}
 
+### Set up your virtual environment
```

Comment:
This doesn't make sense here because following the page flow they'd already have installed the SDK now - let's move this to the requirements section and have it in the python tab where the install command is

Reply:
oh yes thank you 😵‍💫 , will do and comment here when done

Reply:
done!